### PR TITLE
[16.0] [MIG] product_assortment

### PR DESCRIPTION
**Pull request by marielejeune**

### CODE REVIEW

**Review by rousseldenis — APPROVED** [review]
Code review

File: product_assortment/models/ir_filters.py [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+        model = self.env.ref("product.model_product_product")
+        for vals in vals_list:
+            if not vals.get("model_id"):
+                vals.update({"model_id": model.model})
+        return vals_list
+
+    @api.model_create_multi
+    def create(self, vals_list):
+        self._update_assortment_default_values(vals_list=vals_list)
+        return super().create(vals_list=vals_list)
```
Comment (apineux) [inline product_assortment/models/ir_filters.py]: It is clearer if we put the field definitions at the beginning of the class and the methods after

Comment (marielejeune) [reply product_assortment/models/ir_filters.py]: Done

**Review by lmignon — CHANGES_REQUESTED** [review]
@marielejeune The page layout is broken
image

**Review by lmignon — APPROVED** [review]
Thank you @marielejeune :+1: (Code review + functional tests)

The UI is now fixed (tested in OC et OE)

image

**Comment by lmignon** [comment]
ping @rousseldenis

**Comment by rousseldenis** [comment]
@marielejeune @lmignon Could you please fix invalidate_cache() method in test?

https://github.com/OCA/product-attribute/actions/runs/3280669142/jobs/5401736565

**Comment by marielejeune** [comment]
@rousseldenis see https://github.com/OCA/product-attribute/pull/1197

**Pull request merged**